apps/css-workshop/src/pages/index.astro

```diff
@@ -16,9 +17,9 @@ const components = await fs.promises.readdir('./src/pages').then((files) => {
 
   <h1>iTwinUI-CSS</h1>
   <p>
-    This site is specifically designed for testing the CSS from&nbsp;<code class='iui-code'
-      >@itwin/itwinui-css</code
-    >. To learn more about iTwinUI, please visit the&nbsp;<a
+    This site is specifically designed for testing the CSS from&nbsp;<Code_>
+      @itwin/itwinui-css
+    </Code_>. To learn more about iTwinUI, please visit the&nbsp;<a
```

Comment:
Same comment as https://github.com/iTwin/iTwinUI/pull/1980#r1552426134

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1980/commits/352e50ed9bb41417b2ac9f2b046c55c82a3d0bca)